libraries/botbuilder-dialogs-adaptive/src/actions/baseInvokeDialog.ts

```diff
@@ -15,7 +15,7 @@ import { replaceJsonRecursively } from '../jsonExtensions';
  */
 export class BaseInvokeDialog<O extends object = {}> extends Dialog<O> implements DialogDependencies {
     /**
-     * Initializes a new instance of the `BaseInvokeDialog` class.
+     * Initializes a new instance of the [BaseInvokeDialog](xref:botbuilder-dialogs-adaptive.BaseInvokeDialog) class.
      * Expression for `dialogId` to call (allowing dynamic expression).
      * @param dialogIdToCall Optional. Id of the dialog to call.
```

Comment:
Should we use the Dialog class reference here?, more lines with this point on this file.